### DISCUSSION TITLE
Support for external CSS Files, and custom css variables

### DISCUSSION THREAD

**Discussion by mra-miratech** [blackears/svgSalamander#97] [Q&A]
Im curious of Salamander Supports svgs which use CSS Styling via CSS selectors and allows custom properties (CSS variables). E.g
—critical-col: red;
.critical {stroke: var(—critical-col)}
<line class=“critical” …..>

ideally, even with reference from svg to extern al Stylesheets. And with a possibility to set the values of the custom variables from Java Code .

That woukd be fantastic.

**Comment by blackears** [comment]
Unfortunately the engine isn't sophisticated enough to support that.  The CSS engine doesn't support much more than parsing primitive values.
